eth/protocols/snap/sync.go

```diff
@@ -2016,6 +2067,14 @@ func (s *Syncer) processStorageResponse(res *storageResponse) {
 					options = options.WithWriter(func(path []byte, hash common.Hash, blob []byte) {
 						rawdb.WriteTrieNode(batch, owner, path, hash, blob, s.scheme)
 					})
+					if s.scheme == rawdb.PathScheme {
+						options = options.WithCleaner(func(path []byte) {
+							s.cleanPath(batch, owner, path)
+						})
+						// Keep the left boundary as it's the first range.
+						// Skip the right boundary if it's not the last range.
+						options.SkipBoundary(false, r.End() != common.MaxHash, boundaryStorageNodesGauge)
```

Comment:
Here you're missing the `options = ` part. Although both to the same thing, IMO we should be consistent. I'm happy to drop the `options = ` from all call-sites too FWIW, but let's make them consisntent.